README.md

```diff
@@ -1,5 +1,51 @@
-# Mini-Projects
+# Machine Learning Engineering Course Mini-Projects
+
+This is the repository of the current miniprojects for Springboard's Machine Learning Engineering course. As you work through the curriculum, you will need to complete each and everyone of those mini-projects and review and discuss them with your mentor.
+
+It is highly recommended to fork this repository on Github and share it with your mentor so that they can easily follow your progress.
+
+## Installation
+
+Below are details instructions on how to get those mini projects up and running. None of those projects should require tons of CPU or memory, so you are welcome to either run those locally on your computer, or through any hosted jupyter notebook services such as Deepnote or Google's Collab.
+
+### Local
+
+To run those projects locally, we recommend you first install Anaconda for your operating system (Windows, Mac, Linux) by downloading it and following the instructions here: https://www.anaconda.com/products/individual#Downloads
+
+Once anaconda is installed, run the following command to create dedicated python environment with all the libraries and packages needed to run all the mini projects in this repo:
```

Comment:
@arnaudsj a student needs to first make sure they are in this repos root, can you add instruction for them to clone their fork to their local machine?

Reply:
Got it. Added a new paragraph with a link to the excellent Github Doc: https://docs.github.com/en/github/getting-started-with-github/fork-a-repo